swagger.yaml

```diff
@@ -1,42 +1,233 @@
 swagger: "2.0"
+
 info:
   title: dp-search-reindex-api
   description: "Provides details about search reindex jobs and enables creation and running of them"
   version: 1.0.0
   license:
     name: "Open Government Licence v3.0"
     url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+
 basePath: "/v1"
+
 schemes:
   - http
+
 tags:
-  - name: "hello"
+  - name: "search-reindex"
+
 paths:
-  /hello:
+
+  /jobs/{id}:
+    get:
+      description: Get a specific job.
+      parameters:
+        - $ref: '#/parameters/id'
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Job"
+        500:
+          $ref: '#/responses/InternalError'
+
+  /jobs:
     get:
-      tags:
-        - hello
-      summary: Example hello world endpoint
-      description: Returns an example payload.
+      description: Get a list of jobs.
+      parameters:
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+        - $ref: '#/parameters/sort'
+        - $ref: '#/parameters/state'
       produces:
         - application/json
       responses:
         200:
           description: OK
           schema:
-            $ref: "#/definitions/HelloResponse"
+            $ref: "#/definitions/Jobs"
+        500:
+          $ref: '#/responses/InternalError'
+    post:
+      description: Post new job details and receive the new Job object in response.
```

Comment:
We usually add a summary before a description
 ```
    tags:
        - "Public"
      summary: "Returns the latest version of an area"
      description: "Returns an area for a given id"
```

Reply:
now added summaries to all three endpoints